### DISCUSSION TITLE
اضافه کردن محدودیت تعداد دستگاه های متصل به کد های تولید شده برای هر یوزر

### DISCUSSION THREAD

**Discussion by farzaad7** [Gozargah/Marzban#352] [General]
سلام دوستان .. آیا این امکان هست که در کانفیگ مرزبان ، امکان اعمال **محدودیت تعداد دستگاه یا IP متصل** به کدهای تولید شده رو اضافه کرد ؟ گویا الان هر تعداد یوزر که بخوان میتونن از یک کد استفاده کنن ! اگه اشتباه می کنم بگید .. ممنون

**Comment by sinashk** [comment]
امیدوارم این آپشن اضافه بشه

**Comment by roseshayan** [comment]
اگر این مورد اضافه بشه، زمانی که تعداد کاربران شما بالا بره، فشار بسیار زیادی روی سی پی یو خواهد آمد که شما رو مجبور میکنه تعداد کاربران کمتری رو داشته باشید!

**Comment by adora1989** [comment]
اره عالی میشه تنها عیب این پنل همینه
با برنامه نپسترنت میشه کانفیگ رو قفل کرد ولی متاسفانه نپسترنت فعلا از reality پشتیبانی نمیکنه و فقط اخرین ورژن v2ray پشتیبانی میکنه که با اونم نمیشه قفل گذاشت

**Comment by ghost** [comment]
> سلام دوستان .. آیا این امکان هست که در کانفیگ مرزبان ، امکان اعمال **محدودیت تعداد دستگاه یا IP متصل** به کدهای تولید شده رو اضافه کرد ؟ گویا الان هر تعداد یوزر که بخوان میتونن از یک کد استفاده کنن ! اگه اشتباه می کنم بگید .. ممنون

سلام این قابلیت بصورت پیش فرض رو پنل وجود نداره ولی میتونی از اسکریپت [V2IpLimit](https://github.com/houshmand-2005/V2IpLimit) استفاده کنی.
ولی در کل محدودیت آیپی فشار اضافه رو سرور شما میاره و بهتره از محدودیت حجم استفاده کنی.

**Comment by nsa14** [comment]
> سلام دوستان .. آیا این امکان هست که در کانفیگ مرزبان ، امکان اعمال **محدودیت تعداد دستگاه یا IP متصل** به کدهای تولید شده رو اضافه کرد ؟ گویا الان هر تعداد یوزر که بخوان میتونن از یک کد استفاده کنن ! اگه اشتباه می کنم بگید .. ممنون 

بجای محدودیت کاربر ک قبلنا برای اکانت ها گذاشته میشد، بجاش از محدودیت حجم استفاده میشه
اینجوری هرکاربر بخاطر حجم خودشم ک شده نمیاد در اختیار دیگران بده

**Comment by NimaKeshavarz** [comment]
منم این مشکل و دارم و دنبال پنل دیگه ای هستم برای رفع این مشکل. من یوزر و برای پدر و مادر ساختم ولی خیلی ها (نوه ها و فامیل) تو گوشی او نا می رن و استفاده می کنن و این وسط نمی دونم کافیگ های منم برمی دارن یا نه و متاسفانه هیچ کنترلی روش ندارم . حجم مصرفی توسط یوزرها بالا هست و نمی دونم کی استفاده می کنه.
فکر کنم علارغم خوبی های این پنل باید به فکر پنل دیگه باشم.
اگر کسی پنل دیگه ای می شناسه که خوب باشه و بشه کنترل کرد تعداد یوزرهای یک کانفیگ و لطفان بگه
